### ISSUE TITLE
can not start the container

### ISSUE THREAD

**Issue by flodotlan** [dali99/dockerfiles#2]
will starting the container with this command:

sudo docker run -t -i -d dandellion/open-source-ethereum-pool -p "8888:8888" -p "8080:8080" -p "8545:8545" -p "6379:6379"  

and have this error:
admin@test01 / $ sudo docker run -t -i -d dandellion/open-source-ethereum-pool -p "8888:8888" -p "8080:8080" -p "8545:8545" -p "6379:6379"
380979a32ea9927dcff1e1c1e03e393bffbc8b0f7c25d06bdb5035856db20f15
docker: Error response from daemon: rpc error: code = 2 desc = "oci runtime error: exec format error".

**Comment by dali99** [comment]
yeah, it's not done, like at all, sort of given up on it tbh. I can't figure out a way other than actually mounting the geth ipc to communicate, making this project kinda moot. I recommend you just make the services work in a vm or something. Maybe I can make IPC work by mounting or something
